docs/user_manual/libraries.md

```diff
@@ -246,12 +246,12 @@ spreadsheet of library data provided by a collaborator. For more information, se
 
 
 {% assign sub = sub | plus: 1 %}
-{% include userman-heading.md section=page.section sub=sub title="Propagating Libraries to Library Dilutions" %}
+{% include userman-heading.md section=page.section sub=sub title="Propagating Libraries to Library Aliquots" %}
 
-To make dilutions from libraries, go to the Libraries list, check the checkboxes next to the libraries you wish to
-propagate from, then click the "Make Dilutions" button in the toolbar at the top of the table. Further details of
-propagating libraries to dilutions will be discussed in the
-[Library Dilutions section - Propagating Dilutions from Libraries](library_dilutions.html#propagating_dilutions_from_libraries).
+To make library aliquots from libraries, go to the Libraries list, check the checkboxes next to the libraries you wish
+to propagate from, then click the "Make aliquots" button in the toolbar at the top of the table. Further details of
```

Comment:
Wasn't capitalized before as "Make dilutions," and I didn't change that. Capitalization of table buttons isn't consistent though, and it would be good to fix that at some point